warden/warden_daemon/tests/start_and_shutdown_test.rs

```diff
@@ -25,14 +28,17 @@ async fn sig_term_shutdown() {
     assert!(handle.await.unwrap().is_ok());
 }
 
-#[tokio::test]
+#[tokio::test(flavor = "multi_thread")]
 #[ignore]
 async fn sig_int_shutdown() {
     let workdir_path_manager = WorkdirManager::new().await;
     let usock_path = workdir_path_manager
         .get_path()
         .join(format!("usock-{}", Uuid::new_v4()));
-    let cli = common::create_example_cli(usock_path, workdir_path_manager.get_path().to_path_buf());
+    let mut cli =
+        common::create_example_cli(usock_path, workdir_path_manager.get_path().to_path_buf());
+    cli.bridge_ip = IpNet::V4(Ipv4Net::new(Ipv4Addr::new(192, 168, 100, 0), 24).unwrap());
```

Comment:
again this is the so called "network address"

Reply:
Done